library/MedEx/API.php

```diff
@@ -2913,136 +2913,119 @@ public function MedExBank($stage)
+                                <div class="form-group mt-3">
+                                    <label for="new_password"><?php echo xlt('Password'); ?>:</label>
+                                    <i id="pwd_check" name="pwd_check" class="top_right_corner nodisplay text-success fa fa-check"></i>
+                                    <i class="fa top_right_corner fa-question" id="pwd_ico_help" aria-hidden="true" onclick="$('#pwd_help').toggleClass('nodisplay');"></i>
+                                    <input type="password" placeholder="<?php xla('Password'); ?>" id="new_password" name="new_password" class="form-control" required>
+                                    <div id="pwd_help" class="nodisplay signup_help"><?php echo xlt('Secure Password Required') . ": " . xlt('8-12 characters long, including at least one upper case letter, one lower case letter, one number, one special character and no common strings'); ?>...</div>
                                 </div>
-
-                                <div id="myModal" class="modal fade" role="dialog">
-                                  <div class="modal-dialog">
-
-                                    <!-- Modal content-->
-                                    <div class="modal-content">
-                                      <div class="modal-header"  style="background-color: #0d4867;color: #fff;font-weight: 700;">
-                                        <button type="button" class="close" data-dismiss="modal" style="color:#fff;opacity:1;box-shadow:unset !important;">&times;</button>
-                                        <h2 class="modal-title" style="font-weight:600;">Sign-Up Confirmation</h2>
-                                      </div>
-                                      <div class="modal-body" style="padding: 10px 45px;">
-                                        <p>You are opening a secure connection to MedExBank.com.  During this step your EHR will synchronize with the MedEx servers.  <br />
+                                <div class="form-group mt-3">
+                                    <label for="new_rpassword"><?php echo xlt('Repeat'); ?>:</label>
+                                    <i id="pwd_rcheck" name="pwd_rcheck" class="top_right_corner nodisplay text-success fa fa-check"></i>
+                                    <input type="password" placeholder="<?php echo xla('Repeat password'); ?>" id="new_rpassword" name="new_rpassword" class="form-control" required>
+                                    <div id="pwd_rhelp" class="nodisplay signup_help" style=""><?php echo xlt('Passwords do not match.'); ?></div>
+                                </div>
+                            </div>
+                            <div id="ihvread" name="ihvread" class="text-left showReminders">
+                                            <input type="checkbox" class="updated required" name="TERMS_yes" id="TERMS_yes" required>
```

Comment:
All inputs should be slash closed at the end like this:
`<input type="checkbox" class="updated required" name="TERMS_yes" id="TERMS_yes" required />`